src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -902,6 +902,35 @@
           (recur (rt/read-char reader)))
         (str sb)))))
 
+(defn get-data-readers*
+  "Returns the merged data reader mappings."
+  []
+  (reduce (fn [data-readers url+source]
+            (let [url      (aget url+source "url")
+                  source   (aget url+source "source")
+                  mappings (r/read-string source)]
+              (when-not (map? mappings)
+                (throw (ex-info (str "Not a valid data-reader map")
+                                {:url url})))
+              (reduce-kv (fn [data-readers tag fn-sym]
+                           (when-not (symbol? tag)
+                             (throw (ex-info (str "Invalid form in data-reader file")
+                                             {:url  url
+                                              :form tag})))
+                           (when (and (data-readers tag)
+                                      (not= (mappings tag) fn-sym))
+                                (throw (ex-info "Conflicting data-reader mapping"
+                                         {:url      url
+                                          :conflict tag
+                                          :mappings data-readers})))
+                           (assoc data-readers tag (eval fn-sym)))
```

Comment:
I've left it as is for now. You probably don't want spoiled lumo users to complain when using regular cljs :)